base/loading.jl

```diff
@@ -1210,7 +1210,7 @@ end
 
 const MAX_NUM_PRECOMPILE_FILES = 10
 
-function compilecache(pkg::PkgId, path::String)
+function compilecache(pkg::PkgId, path::String, show_errors::Bool = true)
```

Comment:
I think it'd be better to comment that `show_errors` is an "internal" interface for `Pkg.precompile`. It's a bit unclear why we have this from just looking at the code in Base.